docs/gaia/join-mainnet.md

```diff
@@ -0,0 +1,152 @@
+of the Cosmos-SDK to use and details about the genesis file.
+:::
+
+::: warning
+**You need to [install gaia](./installation.md) before you go further**
+:::
+
+## Setting Up a New Node
+
+These instructions are for setting up a brand new full node from scratch.
+
+First, initialize the node and create the necessary config files:
+
+```bash
+gaiad init <your_custom_moniker>
+```
+
+::: warning Note
+Monikers can contain only ASCII characters. Using Unicode characters will render your node unreachable.
+:::
+
+You can edit this `moniker` later, in the `~/.gaiad/config/config.toml` file:
+
+```toml
+# A custom human readable name for this node
+moniker = "<your_custom_moniker>"
+```
+
+You can edit the `~/.gaiad/config/gaiad.toml` file in order to enable the anti spam mechanism and reject incoming transactions with less than the minimum gas prices:
+
+```
+# This is a TOML config file.
+# For more information, see https://github.com/toml-lang/toml
+
+##### main base config options #####
+
+# The minimum gas prices a validator is willing to accept for processing a
+# transaction. A transaction's fees must meet the minimum of any denomination
+# specified in this config (e.g. 10uatom).
+
+minimum-gas-prices = ""
+```
+
+Your full node has been initialized! 
+
+## Genesis & Seeds
+
+### Copy the Genesis File
+
+Fetch the testnet's `genesis.json` file into `gaiad`'s config directory.
+
+```bash
+mkdir -p $HOME/.gaiad/config
+curl https://raw.githubusercontent.com/cosmos/launch/master/latest/genesis.json > $HOME/.gaiad/config/genesis.json
+```
+
+Note we use the `latest` directory in the [launch repo](https://github.com/cosmos/launch) which contains details for the mainnet like the latest version and the genesis file. 
+
+::: tip
+If you want to connect to the public testnet instead, click [here](./join-testnet.md)
+:::
+
+To verify the correctness of the configuration run:
+
+```bash
+gaiad start
+```
+
+### Add Seed Nodes
+
+Your node needs to know how to find peers. You'll need to add healthy seed nodes to `$HOME/.gaiad/config/config.toml`. The [`launch`](https://github.com/cosmos/launch) repo contains links to some seed nodes.
+
+If those seeds aren't working, you can find more seeds and persistent peers on a Cosmos Hub explorer (a list can be found on the [launch page](https://cosmos.network/launch)). 
+
+You can also ask for peers on the [Validators Riot Room](https://riot.im/app/#/room/#cosmos-validators:matrix.org)
+
+For more information on seeds and peers, you can [read this](https://github.com/tendermint/tendermint/blob/develop/docs/tendermint-core/using-tendermint.md#peers).
+
+## A note on gas and fees
+
+::: warning
+On Cosmos Hub mainnet, the accepted denom is `uatom`, where `1atom = 1.000.000uatom`
+:::
+
+Transactions on the Cosmos Hub network need to include a transaction fee in order to be processed. This fee pays for the gas required to run the transaction. The formula is the following:
+
+```
+fees = gas * gasPrices
```

Comment:
Should really be `fees = ceil(gas * gasPrices)` if we're being accurate.

Reply:
y

Reply:
```suggestion
fees = ceil(gas * gasPrices)
```